inflection/tools/dictionary-parser/src/main/resources/org/unicode/wikidata/filter_ru.properties

```diff
@@ -0,0 +1,6 @@
+# Copyright 2025 Unicode Incorporated and others. All rights reserved.
+#
+# These are lexemes that should either be ignored due to irrelevance that can't be easily tagged as irrelevant,
+# or words that are just not that common that should be sorted last in the inflection patterns.
+L128740=omit
```

Comment:
This noun is not typical, and it conflicts with the common pronoun. Remove it for now to deconflict it.